tests/clients/boto3/test_boto3_s3.py

```diff
@@ -19,7 +19,7 @@
 def setup_method():
     """ Clear all spans before a test run """
     tracer.recorder.clear_spans()
-    os.remove(download_target_filename)
+#    os.remove(download_target_filename)
```

Comment:
Is this line still needed?

Reply:
I cleaned up all my changes in the boto3 tests, they were related to a piece of code I was trying and had reverted already, no need of changes there, only formatting changes on boto3 tests